src/test/java/io/vertx/test/redis/RedisClientTLSTest.java

```diff
@@ -149,7 +146,10 @@ public void testConnectionOptions(TestContext should) {
       .onSuccess(conn -> {
         conn.send(Request.cmd(Command.PING))
```

Comment:
I think here we can simply do `onComplete(should.asyncAssertSuccess(res -> ...))` (applies to other parts)

Reply:
I tried to stay away from rewriting this test too much, but I can try :-)

Reply:
Done.